web-bluetooth/read-characteristic-value-changed.js

```diff
@@ -1,13 +1,34 @@
+var bluetoothDevice;
 var batteryLevelCharacteristic;
 
 function onReadBatteryLevelButtonClick() {
-  log('Requesting Bluetooth Device...');
-  navigator.bluetooth.requestDevice(
-    {filters: anyDevice(), optionalServices: ['battery_service']})
-  .then(device => {
-    log('Connecting to GATT Server...');
-    return device.gatt.connect();
+  let requestDevice = Promise.resolve();
```

Comment:
For instance, in this page, now that it's caching the device, we might want another button to explicitly switch devices.